cubic-server/protocol/ClientPackets.hpp

```diff
@@ -101,6 +102,148 @@ struct PingResponse {
 };
 std::shared_ptr<std::vector<uint8_t>> createPingResponse(const PingResponse &);
 
+struct SpawnEntity {
+    int32_t entityId;
+    u128 entityUuid;
+    enum class EntityType : int32_t {
```

Comment:
Are those IDs correct ? Because there is the Camel which is not in 1.19.3 taking an ID, which sounds weird.

Reply:
it is, just locked behind the 1.20 experimental update flag.